src/command/verify/verify.c

```diff
@@ -281,22 +379,24 @@ verifyArchiveInfoFile(void)
 
             // Attempt to load the copy and report on it's status but don't keep it in memory
             const VerifyInfoFile verifyArchiveInfoCopy = verifyInfoFile(
-                INFO_ARCHIVE_PATH_FILE_COPY_STR, false, cfgOptionStrNull(cfgOptRepoCipherPass));
+                INFO_ARCHIVE_PATH_FILE_COPY_STR, false, cfgOptionStrNull(cfgOptRepoCipherPass), errorList);
 
             // If the copy loaded successfully, then check the checksums
             if (verifyArchiveInfoCopy.errorCode == 0)
             {
                 // If the info and info.copy checksums don't match each other than one (or both) of the files could be corrupt so
                 // log a warning but must trust main
                 if (!strEq(verifyArchiveInfo.checksum, verifyArchiveInfoCopy.checksum))
-                    LOG_DETAIL("archive.info.copy does not match archive.info");
+                {
+                    verifyErrorNew(errorList, logLevelDetail, strNewZ("archive.info.copy does not match archive.info"));
+                }
```

Comment:
It appears that the surrounding code doesn't use braces for 1-statement code blocks 
```suggestion
                    verifyErrorNew(errorList, logLevelDetail, strNewZ("archive.info.copy does not match archive.info"));
```